UIKitVK/UIKitVK/Controller/FriendCollectionViewController.swift

```diff
@@ -54,8 +55,34 @@ final class FriendCollectionViewController: UICollectionViewController {
     private func fetchPhotos() {
         networkService.fetchPhotos(ownerID: ownerID) { [weak self] photos in
             guard let self = self else { return }
-            self.photos = photos
-            self.collectionView.reloadData()
+            switch photos {
+            case let .success(data):
+                self.photos = data
+                self.networkService.saveDataRealm(self.photos)
+                self.collectionView.reloadData()
+            case let .failure(error):
+                print("\(Constants.OtherConstants.error): \(error.localizedDescription)")
+            }
+        }
+    }
+
+    private func unloadingPhotosRealm() {
+        do {
+            let realm = try Realm()
+            let friendPhotos = Array(realm.objects(Photo.self))
+            let userID = friendPhotos.map(\.ownerID)
+            if userID.contains(where: { realmID in
+                realmID == ownerID
```

Comment:
```suggestion
                ownerID == ownerID
```
я бы назвал так, ты же не айди реалма сравниваешь а айди овнера